commit 34568c544e480b4c931302e48a8ba9e154f3101e
Author: Maxime Ripard <maxime.ripard@free-electrons.com>
Date:   Mon Jan 23 11:41:47 2017 +0100

    rtc: sun6i: Add some locking
    
    commit a9422a19ce270a22fc520f2278fb7e80c58be508 upstream.
    
    Some registers have a read-modify-write access pattern that are not atomic.
    
    Add some locking to prevent from concurrent accesses.
    
    Acked-by: Chen-Yu Tsai <wens@csie.org>
    Signed-off-by: Maxime Ripard <maxime.ripard@free-electrons.com>
    Signed-off-by: Alexandre Belloni <alexandre.belloni@free-electrons.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1343_34568c544e480b4c931302e48a8ba9e154f3101e.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1343_34568c544e480b4c931302e48a8ba9e154f3101e.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
